go.mod

```diff
@@ -14,14 +14,12 @@ require (
 	github.com/mattn/go-isatty v0.0.9 // indirect
```

Comment:
Maybe we can use this PR to update terraform to `0.12.12` :)

Reply:
done